core/crypto.mk

```diff
@@ -52,6 +52,12 @@ ifeq ($(CFG_CRYPTO_WITH_CE),y)
 
 $(call force,CFG_AES_GCM_TABLE_BASED,n,conflicts with CFG_CRYPTO_WITH_CE)
 
+# CFG_HWSUPP_PMULL_64 defines whether AArch64 supports polynomial multiplies
```

Comment:
`pmull_ghash_update_p64()` can be available in AArch32 too

Reply:
Right. I'll rename/reword the stuff again, then.